### ISSUE TITLE
When editing GradientTexture, inspector interface was blocked by texture

### ISSUE THREAD

**Issue by leaf-node** [godotengine/godot-docs#1999]
https://docs.godotengine.org/en/3.0/getting_started/step_by_step/your_first_game.html#particles

When editing the newly created GradientTexture on a laptop with a 1366x768 pixel screen, the large texture image blocked most of the inspector viewport. I didn't realize that most of it was blocked, so I thought that the interface was broken.

I was able to see the rest of the inspector viewport by resizing the inspector. It might be helpful if you mention the potential need to resize the inspector at this stage of the process.

**Comment by cbscribe** [comment]
Closing as there's been no activity for some time. Feel free to open a new issue if you feel the need is still there.
